CMakeLists.txt

```diff
@@ -97,6 +98,9 @@ if(BUILD_SHARED_LIBS)
   )
 
   if(USE_TPM)
+    target_link_libraries(xtt PUBLIC tss2::sys)
+    target_link_libraries(xtt PUBLIC tss2::tcti-device)
+    target_link_libraries(xtt PUBLIC tss2::tcti-mssim)
```

Comment:
Should/can the `tcti-msssim` be put behind a config option?  It isn't needed in production environments (e.g., our partner gateways) and might be difficult to compile or obtain.

Reply:
Doing so would require more than gating this `target_link_libraries` call, it would require changes in the code to gate all the usage of the mssim (in the initialization of the TCTI). Not a huge effort, but not trivial

The xtt code was written when the xaptum-tpm library was monolithic, so all the functionality was assumed available. We just have the `USE_TPM` ifdef's, so the ifdef's might get a little ugly

The two ways a platform would obtain the TSS would be xaptum-tpm or tpm2-software, right? And both of those install a `libtss2-tcti-mssim`

Reply:
> The two ways a platform would obtain the TSS would be xaptum-tpm or tpm2-software, right? And both of those install a libtss2-tcti-mssim

Ok, that makes sense.